static/styles/settings.css

```diff
@@ -1667,6 +1667,26 @@ $option_title_width: 180px;
     margin-bottom: 20px;
 }
 
+.settings_textarea {
+    color: hsl(0, 0%, 33%);
+    background-color: hsl(0, 0%, 100%);
```

Comment:
The dark theme CSS are applied as expected for all textarea elements except the one in invite modal because that used ID. Changed it to instead use a class. Now the dark theme CSS takes precedence since that also uses `dark_theme` class for all textarea elements.

Reply:
Excellent.